db/connection.js

```diff
@@ -1,5 +1,5 @@
-const { Pool } = require('pg');
-const ENV = process.env.NODE_ENV || 'development';
+const { Pool } = require('pg'); 
+const ENV = process.env.NODE_ENV || 'test';
```

Comment:
This is the correct one: const { Pool } = require('pg');
const ENV = process.env.NODE_ENV || 'development';

Change 'test' with 'development' please 🙂